versions.props

```diff
@@ -36,12 +36,12 @@ com.palantir.tracing:* = 4.23.0
 com.palantir.tritium:* = 0.22.0
 com.squareup:javapoet = 1.13.0
 com.squareup.okio:okio = 2.10.0
-com.zaxxer:HikariCP = 4.0.3
-commons-io:commons-io = 2.10.0
+com.zaxxer:HikariCP = 5.0.0
```

Comment:
Release logs for this is not trivial, so I'm not sure what this major version change means for us.

Reply:
I've reviewed the diff. The primary change is moving to Java 11, and there are some minor improvements to their testing infrastructure. I believe this change is OK.